### PR TITLE
OpenApi compatibility Gemfire 1.15

### PR DESCRIPTION
**Pull request by jubenitezg**
Fix missing information in the new format for POST and PUT endpoints.

### CODE REVIEW

**Review by jinmeiliao — APPROVED** [review]
Thanks!

**Comment by onichols-pivotal** [comment]
Does this change handle both new (`x-jqFilter`) and old (`jqFilter`) field name?  I don't think any of the existing tests catch this, because all the tests of jqFilter use hardcoded sample data (swagger style, not openapi).  Maybe should duplicate each of those tests to add equivalents that test against openapi v3 sample data.

For production, I believe the plugin tolerates the lack of `jqFilter` in order to retain compatibility with very old versions of GemFire that didn't supply it.  However this field is vital to presenting a nice human-readable view of the returned data for many commands.

File: impl/common/endpoints.go [hunk 1]
```diff
@@ -38,6 +56,7 @@ func GetEndPoints(commandData *domain.CommandData, processRequest impl.RequestHe
 		commandData.ConnnectionData.LocatorAddress + "/management/",
 		commandData.ConnnectionData.LocatorAddress + "/management/v1/api-docs",
 		commandData.ConnnectionData.LocatorAddress + "/management/experimental/api-docs",
+		commandData.ConnnectionData.LocatorAddress + "/management/v3/api-docs",
```
Comment (onichols-pivotal) [inline impl/common/endpoints.go]: if these are tried in order, as an optimization maybe the most likely ones should be tried first? e.g. v3, then v1, then the others

File: domain/structs.go [hunk 2]
```diff
@@ -85,10 +86,17 @@ type APIInfo struct {
 
 // RestAPIDetail provides details about an endpoint
 type RestAPIDetail struct {
-	CommandName string         `json:"summary"`
-	JQFilter    string         `json:"jqFilter"`
-	Consumes    []string       `json:"consumes"`
-	Parameters  []RestAPIParam `json:"parameters"`
+	CommandName string          `json:"summary"`
+	JQFilter    string          `json:"jqFilter"`
```
Comment (jmelchio) [inline domain/structs.go:90]: You will need the ability to also parse `x-jqFilter` for the changes made in the newer API. I assume you can simply add an element here and do something similar to what you have done for `Definitions` and `Components`.

**Review by jmelchio — CHANGES_REQUESTED** [review]
Looks good, just needs `x-jqFilter` json element support. Apologies for not bringing that up ahead of time.

**Review by jmelchio — APPROVED** [review]
LGTM

**Review by onichols-pivotal — APPROVED** [review]

**Pull request merged**